db/migrate/20151027130308_create_evidences.rb

```diff
@@ -0,0 +1,10 @@
+class CreateEvidences < ActiveRecord::Migration
+  def change
+    create_table :evidences do |t|
+      t.belongs_to :user, index: true
+      t.belongs_to :requirement, index: true
+      t.datetime :valid_until, null: false
```

Comment:
shouldn't the requirement take care of validity? i.e. a requirement with `within_months` just selects all evidence that is created during that timeframe in order to check whether it is met...